src/boot/efi/meson.build

```diff
@@ -197,18 +197,19 @@ if have_gnu_efi
         compile_args = cc.get_supported_arguments(
                 basic_disabled_warnings +
                 possible_common_cc_flags + [
-                        '-ffreestanding',
                         '-fno-stack-protector',
                         '-fno-strict-aliasing',
                         '-fpic',
-                        '-fshort-wchar',
+                        '-fwide-exec-charset=UCS2',
```

Comment:
are you sure about this? Windows uses UTF16, not UCS2. Are you sure EFI uses UCS2?

Reply:
some googling revealed you are apparently right. Crazy shit.

Reply:
I haven't even seen that section 😲. I definitely have to try that out sometime. I did see the font section before and was wondering if that could be leveraged to provide our own font. And more importantly all the important glyphs, since vendors are only required to provide latin1 and some box drawing glyphs.

Reply:
Which spec version? I'm looking at Version 2.7 Errata A, and there is not 33.2.6.2.4…

Reply:
https://uefi.org/sites/default/files/resources/UEFI_Spec_2_9_2021_03_18.pdf